### ISSUE TITLE
Atom shows error when opening script

### ISSUE THREAD

**Issue by dsyme** [ionide/ionide-fsharp#184]
I get this error when opening any script in the HandsOnTutorial directory of https://github.com/mbraceproject/MBrace.StarterKit/ in Atom 

![capture](https://cloud.githubusercontent.com/assets/7204669/11429723/12eb8064-9474-11e5-9dc7-cb12023b5b1c.JPG)

I can debug it with some help. Is there an easy way to get at the FSHarp.Compiler.Services ProjectOptions being used by Ionide, or other information being passed to FsAutoComplete.exe?

**Comment by Krzysztof-Cieslak** [comment]
I think the easiest way to debug it is to test FsAutoComplete.exe. It looks like it's some problem in core FSAC so I guess You can just debug FSAC console version and it should be reproduced. Ionide has no direct access to ProjectOptions, it's all handled internally by FSAC.

@cc: rneatherway

**Comment by dsyme** [comment]
@Krzysztof-Cieslak Ionide must be starting FsAutoComplete.exe and transacting information with it?  Is there some way I can get a log of what's happening?

**Comment by Krzysztof-Cieslak** [comment]
Yes, Ionide is starting FSAC process (Suave version in this case, but command line interface provides same functionality) and is sending requests to it (in this case it's `parse` request - it sends file name and content of file to FSAC and gets list of errors back - example of using it with command line is presented in FSAC readme).

In Ionide-fsharp plugin options there is possibility of turning on "Debug Mode" which saves all communication to `.ionide.debug` file (restarting Atom may be required)

But to be fair, I'm pretty sure it's problem on FSAC side, not Ionide or communication between those 2.
CC: @rneatherway

**Comment by rneatherway** [comment]
Debug output follows. The first set of options is the results returned by FCS, the second after the "correct" FSharp.Core is ensured. This changes it from 4.3.1.0 to 4.4.0.0. However, leaving the assemblies alone doesn't make any difference -- the error still appears.

```
{ProjectFileName =
  "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\0-provision-azure-cluster.fsx.fsproj";
 ProjectFileNames =
  [|"C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\MBrace.Azure.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\MBrace.Azure.Management.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\AzureCluster.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\0-provision-azure-cluster.fsx"|];
 OtherOptions =
  [|"--noframework"; "--warn:3";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Numerics.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\mscorlib.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\FSharp\.NETFramework\v4.0\4.3.1.0\FSharp.Core.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Xml.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Runtime.Remoting.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Runtime.Serialization.Formatters.Soap.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Data.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Drawing.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Core.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Web.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Web.Services.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Windows.Forms.dll";
    "-r:C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\FSharp.Compiler.Interactive.Settings.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/Streams/lib/net45/Streams.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/MBrace.Flow/lib/net45/MBrace.Flow.dll";
    "-r:C:\Windows\Microsoft.Net\assembly\GAC_MSIL\System.Runtime\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Runtime.dll";
    "-r:C:\Windows\Microsoft.Net\assembly\GAC_MSIL\System.Threading.Tasks\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Threading.Tasks.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Hyak.Common\lib\net40\Hyak.Common.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Bcl.Async\lib\net40\Microsoft.Threading.Tasks.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Azure.Common\lib\net40\Microsoft.Azure.Common.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Azure.Common\lib\net40\Microsoft.Azure.Common.NetFramework.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management\lib\net40\Microsoft.WindowsAzure.Management.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.Compute\lib\net40\Microsoft.WindowsAzure.Management.Compute.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.Storage\lib\net40\Microsoft.WindowsAzure.Management.Storage.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.ServiceBus\lib\net40\Microsoft.WindowsAzure.Management.ServiceBus.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\lib\net45\MBrace.Azure.Management.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/MBrace.Azure/tools\MBrace.Runtime.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Newtonsoft.Json.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\FsPickler.Json.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\FsPickler.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\System.Spatial.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Mono.Cecil.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Vagabond.AssemblyParser.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Vagabond.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Argu.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.Edm.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.OData.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.Services.Client.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.ServiceBus.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.WindowsAzure.Configuration.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.WindowsAzure.Storage.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Core.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Runtime.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Azure.dll"|];
 ReferencedProjects = [||];
 IsIncompleteTypeCheckEnvironment = false;
 UseScriptResolutionRules = true;
 LoadTime = 31/12/9999 23:59:59;
 UnresolvedReferences =
  Some Microsoft.FSharp.Compiler.SourceCodeServices.UnresolvedReferencesSet;}
{ProjectFileName =
  "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\0-provision-azure-cluster.fsx.fsproj";
 ProjectFileNames =
  [|"C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\MBrace.Azure.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\MBrace.Azure.Management.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\AzureCluster.fsx";
    "C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\0-provision-azure-cluster.fsx"|];
 OtherOptions =
  [|"-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\FSharp\.NETFramework\v4.0\4.4.0.0\FSharp.Core.dll";
    "--noframework"; "--warn:3";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Numerics.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\mscorlib.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Xml.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Runtime.Remoting.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Runtime.Serialization.Formatters.Soap.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Data.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Drawing.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Core.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Web.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Web.Services.dll";
    "-r:C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.Windows.Forms.dll";
    "-r:C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\FSharp.Compiler.Interactive.Settings.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/Streams/lib/net45/Streams.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/MBrace.Flow/lib/net45/MBrace.Flow.dll";
    "-r:C:\Windows\Microsoft.Net\assembly\GAC_MSIL\System.Runtime\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Runtime.dll";
    "-r:C:\Windows\Microsoft.Net\assembly\GAC_MSIL\System.Threading.Tasks\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Threading.Tasks.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Hyak.Common\lib\net40\Hyak.Common.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Bcl.Async\lib\net40\Microsoft.Threading.Tasks.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Azure.Common\lib\net40\Microsoft.Azure.Common.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.Azure.Common\lib\net40\Microsoft.Azure.Common.NetFramework.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management\lib\net40\Microsoft.WindowsAzure.Management.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.Compute\lib\net40\Microsoft.WindowsAzure.Management.Compute.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.Storage\lib\net40\Microsoft.WindowsAzure.Management.Storage.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\..\Microsoft.WindowsAzure.Management.ServiceBus\lib\net40\Microsoft.WindowsAzure.Management.ServiceBus.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure.Management\lib\net45\MBrace.Azure.Management.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\HandsOnTutorial\../packages/MBrace.Azure/tools\MBrace.Runtime.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Newtonsoft.Json.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\FsPickler.Json.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\FsPickler.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\System.Spatial.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Mono.Cecil.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Vagabond.AssemblyParser.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Vagabond.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Argu.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.Edm.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.OData.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.Data.Services.Client.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.ServiceBus.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.WindowsAzure.Configuration.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\Microsoft.WindowsAzure.Storage.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Core.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Runtime.dll";
    "-r:C:\Users\robin\dev\MBrace.StarterKit\packages\MBrace.Azure\tools\MBrace.Azure.dll"|];
 ReferencedProjects = [||];
 IsIncompleteTypeCheckEnvironment = false;
 UseScriptResolutionRules = true;
 LoadTime = 31/12/9999 23:59:59;
 UnresolvedReferences =
  Some Microsoft.FSharp.Compiler.SourceCodeServices.UnresolvedReferencesSet;}
{"Kind":"info","Data":"Background parsing started"}
{"Kind":"errors","Data":[{"FileName":"C:\\Users\\robin\\dev\\MBrace.StarterKit\\HandsOnTutorial\\0-provision-azure-cluster.fsx","StartLine":1,"EndLine":1,"StartColumn":1,"EndColumn":81,"Severity":"Error","Message":"The module/namespace 'System.Runtime.CompilerServices' from compilation unit 'mscorlib' did not contain the namespace, module or type 'ExtensionAttribute'","Subcategory":"typecheck"}]}
```

**Comment by Krzysztof-Cieslak** [comment]
This was fixed in `1.4.12`
